pkg/generic/thrift/base.go

```diff
@@ -52,52 +52,52 @@ func (p *TrafficEnv) SetEnv(val string) {
 	p.Env = val
 }
 
-var fieldIDToName_TrafficEnv = map[int16]string{
+var fieldIDToNameTrafficEnv = map[int16]string{
```

Comment:
this file is generated by thriftgo, and it's unnecessary to lint auto-generated code. To prevent some lint warnings, it's resonable to add a header like `// Code generated by thriftgo (0.0.25). DO NOT EDIT.` to this file.